components/tip/stories/tip.stories.js

```diff
@@ -0,0 +1,31 @@
+import '@vonage/vwc-tip';
+import { html } from 'lit-element';
+import { spread } from '@open-wc/lit-helpers';
+import { argTypes } from './arg-types.js';
+
+export default {
+	title: 'Alpha/Components/tooltip',
```

Comment:
if we stick with tip as the leading component name, this should be tip
```suggestion
	title: 'Alpha/Components/tip',
```

Reply:
@yinonov I don't think anyone would look up the word tip in the storybook if he wanted to add a tooltip to his project.

Reply:
it's a case but not that strong of a case. a learning curve is always needed, searching with the search input will still find it. and what is the main module? tooltip or toggle/popup tip?

Reply:
We have vwc-tooltip and vwc-tip